packages/sitecore-jss-nextjs/src/editing/editing-render-middleware.ts

```diff
@@ -352,7 +350,7 @@ export class MetadataHandler {
         itemId: query.sc_itemid,
         language: query.sc_lang,
         // sc_variant is an array in the query params, but we only need the first value
-        variantId: query.sc_variant.split(',')[0],
+        variantId: query.sc_variant?.split(',')[0] || '',
```

Comment:
Just realized we still might need to have another unit test to cover previewData default parameters. Like it's done [here](https://github.com/Sitecore/jss/pull/1835/files#diff-12306968a1fa59bd9910f11499c4ebb7ac3f933938fe68e6376aabde1abe52abR225)
You can add unit test called: ''should handle request with missing optional parameters". See [unit test example](https://github.com/Sitecore/jss/pull/1835/files#diff-12306968a1fa59bd9910f11499c4ebb7ac3f933938fe68e6376aabde1abe52abR184)